cmd/api/main.go

```diff
@@ -349,7 +422,6 @@ func main() {
 	err = migrationUp()
 	if err != nil && !errors.Is(err, migrate.ErrNoChange) {
 		log.Println(err)
-		os.Exit(1)
```

Comment:
Are you sure you want to continue running the tests after having a migration error?